org_fedora_oscap/gui/spokes/oscap.py

```diff
@@ -581,18 +583,13 @@ def _update_profiles_store(self):
             # not initialized, cannot do anything
             return
 
-        if self._using_ds and self._ds_checklists is None:
+        if self._using_ds and (
+                self._current_xccdf_id is None or self._current_ds_id is None):
             # not initialized, cannot do anything
             return
 
         self._profiles_store.clear()
-
-        if self._using_ds:
-            profiles = self._content_handler.get_profiles(self._current_ds_id,
-                                                          self._current_xccdf_id)
-        else:
-            # pylint: disable-msg=E1103
-            profiles = self._content_handler.profiles
+        profiles = self._content_handler.get_profiles()
```

Comment:
This can lead to exceptions, so what about dispatching a logging message that something went wrong with the update of profiles, and pass the exception on?
The previous behavior was equally bad about this, and any exception would probably kill the addon, so I wouldn't concentrate on this aspect in this PR.